src/OpenLoco/GameState.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include "Company.h"
+#include "Entities/Entity.h"
+#include "Industry.h"
+#include "Map/Animation.h"
+#include "Map/Wave.h"
+#include "Station.h"
+#include "Town.h"
+
+namespace OpenLoco
+{
+#pragma pack(push, 1)
+    struct GameState
+    {
+        Utility::prng rng;                // 0x000000 (0x00525E18)
+        uint32_t pad_0008[3];             // 0x000008
+        uint32_t currentDay;              // 0x000014 (0x00525E2C)
+        uint16_t dayCounter;              // 0x000018
+        uint16_t currentYear;             // 0x00001A
+        uint8_t currentMonth;             // 0x00001C
+        uint8_t currentDayOfMonth;        // 0x00001D
+        int16_t savedViewX;               // 0x00001E
+        int16_t savedViewY;               // 0x000020
+        uint8_t savedViewZoom;            // 0x000022
+        uint8_t savedViewRotation;        // 0x000023
+        uint8_t playerCompanyId;          // 0x000024
+        uint8_t pad_0025[0x146 - 0x25];   // 0x000025
+        uint32_t scenarioTicks;           // 0x000146 (0x00525F5E)
+        uint16_t pad_014A;                // 0x00014A (0x00525F62)
+        uint32_t scenarioTicks2;          // 0x00014C (0x00525F64)
+        uint32_t magicNumber;             // 0x000150 (0x00525F68)
+        uint16_t numMapAnimations;        // 0x000154 (0x00525F6C)
+        uint8_t pad_0156[0x02BC - 0x156]; // 0x000156
+        char scenarioName[64];            // 0x0002BC (0x005260D4)
+        char scenarioDetails[256];        // 0x0002FC (0x00526114)
+        uint8_t pad_03FC[0x434 - 0x3FC];  // 0x0003FC
+        uint16_t fixFlags;                // 0x000434 (0x0052624C)
+        uint8_t pad_0436[0xB96C - 0x436]; // 0x0003FC
+        Company companies[15];            // 0x00B96C (0x00531784)
```

Comment:
Suggest all limits are moved to a standalone header Limits.h within namespace `OpenLoco::Limits` then we can have an `S5::Limits` as well which will show the relationship and which ones are in the future different.

Reply:
That's fine with me.